src/Http/Requests/JobSiteVisitRequest.php

```diff
@@ -0,0 +1,38 @@
+<?php
+
+namespace Xguard\Coordinator\Http\Requests;
+
+use Xguard\Coordinator\Models\JobSiteVisit;
+
+class JobSiteVisitRequest extends BaseFormRequest
+{
+    const SUPERVISOR_SHIFT_ID_REQUIRED = 'supervisor_shift_id.required';
+    const JOB_SITE_ID_REQUIRED ='job_site_id.required';
+    const START_TIME_REQUIRED = 'start_time.required';
+    const END_TIME_REQUIRED = 'end_time.required';
+
+    public function authorize(): bool
```

Comment:
Use trait to avoid repeating  this bloc of code 

public function authorize(): bool
    {
        return true;
    }